### ISSUE TITLE
[Device Support Request] Various HUE & Paulmann devices

### ISSUE THREAD

**Issue by popy2k14** [zigpy/zha-device-handlers#1743]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
Infos about if the following devices are compatible with ZHA,
and if not, it's an device request.

<details>

<summary>Device signature</summary>

those are just device id's from deconz API, hope this helps to identify the devices
```
LCT015
LTW012
LTW010
LCT007
LTW013
LWG004
LCL001
500.48
500.43 Switch Controller (Paulmann)
```
</details>

**Additional context**
Planing to move from deconz to ZHA, and want to check compatibility first.
Already checked: https://zigbee.blakadder.com/zha.html and removed the compatible ones from the list.

**Comment by kryanth** [comment]
The Philips Hue devices (LCT*, LWT*, LTW*) just work. 

LCL001 is listed on blakadder as working, it probably just does.

**Comment by popy2k14** [comment]
ok, thx.
ANy hints about the paulmann devices:

500.48
500.43 Switch Controller (Paulmann)

?

**Comment by jerrm** [comment]
If you have more than a handful of devices, do yourself a favor and get a second coordinator.  Keep both running in parallel and move devices at your leisure instead of a rushing to do it all at once.

**Comment by popy2k14** [comment]
thx for the hint.
already don this way and got an sonoff 3.0 dongle plus.

**Comment by TheJulianJES** [comment]
Like mentioned, basically all Hue devices are supported.
No idea about the Paulmann devices. If they follow standards, they'll work.

If you have issues with those devices, open a new issue and provide the requested information (for which you need to have the device in ZHA).
